### DISCUSSION TITLE
How do I exclude a drive from being scanned?

### DISCUSSION THREAD

**Discussion by MisterGort** [Klocman/Bulk-Crap-Uninstaller#511] [Q&A] [answered]
I'm doing cleanup on my computer, and I want to exclude my gaming drive from BCU. How do I do that? Thanks!

**Comment by Klocman** [comment, marked as answer]
You can try to deselect scanning removable drives in `Settings > Folders` or both options if it's not a removable drive. I suggest adding an exclude search filter for that drive instead, see Advanced filtering in the readme.
